torchtitan/tools/metrics.py

```diff
@@ -96,6 +96,11 @@ def build_device_memory_monitor():
     return device_memory_monitor
 
 
+def metric_processor(metrics: Dict[str, Any]):
```

Comment:
hmm.  maybe we can do this offline in a util if we want to rename metrics?  (i assume that's what this is for?)

Reply:
Well this is to update it offline but we need a hook in train.py.